File: 0088-merge-sorted-array/0088-merge-sorted-array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
       nums1.insert(nums1.begin() + m, nums2.begin(), nums2.end());
       nums1.resize(m + n);
       std::sort(nums1.begin(), nums1.end());
    }
};

File: 0088-merge-sorted-array/0088-merge-sorted-array.rs
impl Solution {
    pub fn merge(nums1: &mut Vec<i32>, m: i32, nums2: &mut Vec<i32>, n: i32) {
        nums1.splice(m as usize..(m + n) as usize, nums2.iter().cloned());
        
        // Sort nums1 in place
        nums1.sort();
    }
}